tests/apollo/test_skvbc_backup_restore.py

```diff
@@ -80,10 +80,10 @@ async def test_checkpoint_propagation_after_restarting_replicas(self, bft_networ
             verify_checkpoint_persistency=False
         )
 
-        # stop n replicas in a random order with a delay of 10s in between
+        # stop n replicas in a random order with a delay of 5s in between
```

Comment:
Was it sensitive to the time? What was the reason to change the delay?

Reply:
If replicas are stopped with 10 seconds delay in between, they can start view change while stopping after the view change timer expires.